get_us_midlatitude_cyclone_abi in satpy.demo fails

**Describe the bug**

`demo.get_us_midlatitude_cyclone_abi()` downloads more files than expected.

**To Reproduce**

```python
from satpy import Scene, demo
from satpy.utils import debug_on; debug_on()

filenames = demo.get_us_midlatitude_cyclone_abi()
```

**Expected behavior**

Return fine (or downloads <= 16 files)

**Actual results**

Creates the folder and downloads 26 files.

```
[INFO: 2020-08-02 16:44:18 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C01_G16_s20190730002190_e20190730004563_c20190730005006.nc
[DEBUG: 2020-08-02 16:44:18 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C01_G16_s20190730002190_e20190730004563_c20190730005006.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:44:19 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C01_G16_s20190730002190_e20190730004563_c20190730005006.nc?generation=1552521915168655&alt=media HTTP/1.1" 206 5205446
[DEBUG: 2020-08-02 16:44:21 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C01_G16_s20190730002190_e20190730004563_c20190730005006.nc> read: 4194304 - 8388608
[DEBUG: 2020-08-02 16:44:21 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C01_G16_s20190730002190_e20190730004563_c20190730005006.nc> read: 5205446 - 9399750
[INFO: 2020-08-02 16:44:21 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc
[DEBUG: 2020-08-02 16:44:21 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:44:21 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc?generation=1552521932812007&alt=media HTTP/1.1" 206 9437184
[DEBUG: 2020-08-02 16:44:25 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 4194304 - 8388608
[DEBUG: 2020-08-02 16:44:25 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 8388608 - 12582912
[DEBUG: 2020-08-02 16:44:25 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc?generation=1552521932812007&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:44:28 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 12582912 - 16777216
[DEBUG: 2020-08-02 16:44:28 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 16777216 - 20971520
[DEBUG: 2020-08-02 16:44:28 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc?generation=1552521932812007&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:44:32 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 20971520 - 25165824
[DEBUG: 2020-08-02 16:44:32 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 25165824 - 29360128
[DEBUG: 2020-08-02 16:44:32 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc?generation=1552521932812007&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:44:35 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 29360128 - 33554432
[DEBUG: 2020-08-02 16:44:35 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 33554432 - 37748736
[DEBUG: 2020-08-02 16:44:35 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc?generation=1552521932812007&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:44:39 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 37748736 - 41943040
[DEBUG: 2020-08-02 16:44:39 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 41943040 - 46137344
[DEBUG: 2020-08-02 16:44:39 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc?generation=1552521932812007&alt=media HTTP/1.1" 206 7422037
[DEBUG: 2020-08-02 16:44:42 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 46137344 - 50331648
[DEBUG: 2020-08-02 16:44:42 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 50331648 - 54525952
[DEBUG: 2020-08-02 16:44:42 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730002190_e20190730004563_c20190730005003.nc> read: 50413653 - 54607957
[INFO: 2020-08-02 16:44:42 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc
[DEBUG: 2020-08-02 16:44:42 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:44:42 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc?generation=1552522229997559&alt=media HTTP/1.1" 206 9437184
[DEBUG: 2020-08-02 16:44:46 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 4194304 - 8388608
[DEBUG: 2020-08-02 16:44:46 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 8388608 - 12582912
[DEBUG: 2020-08-02 16:44:46 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc?generation=1552522229997559&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:44:49 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 12582912 - 16777216
[DEBUG: 2020-08-02 16:44:49 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 16777216 - 20971520
[DEBUG: 2020-08-02 16:44:49 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc?generation=1552522229997559&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:44:53 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 20971520 - 25165824
[DEBUG: 2020-08-02 16:44:53 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 25165824 - 29360128
[DEBUG: 2020-08-02 16:44:53 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc?generation=1552522229997559&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:44:56 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 29360128 - 33554432
[DEBUG: 2020-08-02 16:44:56 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 33554432 - 37748736
[DEBUG: 2020-08-02 16:44:56 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc?generation=1552522229997559&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:45:00 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 37748736 - 41943040
[DEBUG: 2020-08-02 16:45:00 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 41943040 - 46137344
[DEBUG: 2020-08-02 16:45:00 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc?generation=1552522229997559&alt=media HTTP/1.1" 206 6806957
[DEBUG: 2020-08-02 16:45:02 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 46137344 - 50331648
[DEBUG: 2020-08-02 16:45:02 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730007190_e20190730009563_c20190730010002.nc> read: 49798573 - 53992877
[INFO: 2020-08-02 16:45:02 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc
[DEBUG: 2020-08-02 16:45:02 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:03 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc?generation=1552522827637283&alt=media HTTP/1.1" 206 9437184
[DEBUG: 2020-08-02 16:45:06 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 4194304 - 8388608
[DEBUG: 2020-08-02 16:45:06 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 8388608 - 12582912
[DEBUG: 2020-08-02 16:45:07 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc?generation=1552522827637283&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:45:10 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 12582912 - 16777216
[DEBUG: 2020-08-02 16:45:10 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 16777216 - 20971520
[DEBUG: 2020-08-02 16:45:10 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc?generation=1552522827637283&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:45:13 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 20971520 - 25165824
[DEBUG: 2020-08-02 16:45:13 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 25165824 - 29360128
[DEBUG: 2020-08-02 16:45:13 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc?generation=1552522827637283&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:45:17 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 29360128 - 33554432
[DEBUG: 2020-08-02 16:45:17 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 33554432 - 37748736
[DEBUG: 2020-08-02 16:45:17 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc?generation=1552522827637283&alt=media HTTP/1.1" 206 8388608
[DEBUG: 2020-08-02 16:45:20 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 37748736 - 41943040
[DEBUG: 2020-08-02 16:45:20 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 41943040 - 46137344
[DEBUG: 2020-08-02 16:45:20 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc?generation=1552522827637283&alt=media HTTP/1.1" 206 5612027
[DEBUG: 2020-08-02 16:45:22 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 46137344 - 50331648
[DEBUG: 2020-08-02 16:45:22 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C02_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 48603643 - 52797947
[INFO: 2020-08-02 16:45:22 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C03_G16_s20190730002190_e20190730004563_c20190730005006.nc
[DEBUG: 2020-08-02 16:45:22 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C03_G16_s20190730002190_e20190730004563_c20190730005006.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:23 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C03_G16_s20190730002190_e20190730004563_c20190730005006.nc?generation=1552521915512040&alt=media HTTP/1.1" 206 7759939
[DEBUG: 2020-08-02 16:45:26 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C03_G16_s20190730002190_e20190730004563_c20190730005006.nc> read: 4194304 - 8388608
[DEBUG: 2020-08-02 16:45:26 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C03_G16_s20190730002190_e20190730004563_c20190730005006.nc> read: 7759939 - 11954243
[INFO: 2020-08-02 16:45:26 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C04_G16_s20190730002190_e20190730004563_c20190730005007.nc
[DEBUG: 2020-08-02 16:45:26 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C04_G16_s20190730002190_e20190730004563_c20190730005007.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:26 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C04_G16_s20190730002190_e20190730004563_c20190730005007.nc?generation=1552521914139539&alt=media HTTP/1.1" 206 1870310
[DEBUG: 2020-08-02 16:45:27 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C04_G16_s20190730002190_e20190730004563_c20190730005007.nc> read: 1870310 - 6064614
[INFO: 2020-08-02 16:45:27 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C05_G16_s20190730002190_e20190730004563_c20190730005015.nc
[DEBUG: 2020-08-02 16:45:27 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C05_G16_s20190730002190_e20190730004563_c20190730005015.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:27 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C05_G16_s20190730002190_e20190730004563_c20190730005015.nc?generation=1552521914943316&alt=media HTTP/1.1" 206 8351134
[DEBUG: 2020-08-02 16:45:30 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C05_G16_s20190730002190_e20190730004563_c20190730005015.nc> read: 4194304 - 8388608
[DEBUG: 2020-08-02 16:45:30 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C05_G16_s20190730002190_e20190730004563_c20190730005015.nc> read: 8351134 - 12545438
[INFO: 2020-08-02 16:45:30 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C06_G16_s20190730002190_e20190730004569_c20190730005009.nc
[DEBUG: 2020-08-02 16:45:30 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C06_G16_s20190730002190_e20190730004569_c20190730005009.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:30 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C06_G16_s20190730002190_e20190730004569_c20190730005009.nc?generation=1552521930181536&alt=media HTTP/1.1" 206 1716251
[DEBUG: 2020-08-02 16:45:31 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C06_G16_s20190730002190_e20190730004569_c20190730005009.nc> read: 1716251 - 5910555
[INFO: 2020-08-02 16:45:31 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C06_G16_s20190730057190_e20190730059569_c20190730100002.nc
[DEBUG: 2020-08-02 16:45:31 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C06_G16_s20190730057190_e20190730059569_c20190730100002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:31 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C06_G16_s20190730057190_e20190730059569_c20190730100002.nc?generation=1552525220819854&alt=media HTTP/1.1" 206 1176522
[DEBUG: 2020-08-02 16:45:32 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C06_G16_s20190730057190_e20190730059569_c20190730100002.nc> read: 1176522 - 5370826
[INFO: 2020-08-02 16:45:32 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C07_G16_s20190730002190_e20190730004574_c20190730005011.nc
[DEBUG: 2020-08-02 16:45:32 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C07_G16_s20190730002190_e20190730004574_c20190730005011.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:32 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C07_G16_s20190730002190_e20190730004574_c20190730005011.nc?generation=1552521929194925&alt=media HTTP/1.1" 206 4037431
[DEBUG: 2020-08-02 16:45:34 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C07_G16_s20190730002190_e20190730004574_c20190730005011.nc> read: 4037431 - 8231735
[INFO: 2020-08-02 16:45:34 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C08_G16_s20190730002190_e20190730004563_c20190730005002.nc
[DEBUG: 2020-08-02 16:45:34 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C08_G16_s20190730002190_e20190730004563_c20190730005002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:34 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C08_G16_s20190730002190_e20190730004563_c20190730005002.nc?generation=1552521927156290&alt=media HTTP/1.1" 206 3287600
[DEBUG: 2020-08-02 16:45:35 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C08_G16_s20190730002190_e20190730004563_c20190730005002.nc> read: 3287600 - 7481904
[INFO: 2020-08-02 16:45:35 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C08_G16_s20190730017190_e20190730019563_c20190730020002.nc
[DEBUG: 2020-08-02 16:45:35 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C08_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:36 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C08_G16_s20190730017190_e20190730019563_c20190730020002.nc?generation=1552522829129020&alt=media HTTP/1.1" 206 3286409
[DEBUG: 2020-08-02 16:45:37 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C08_G16_s20190730017190_e20190730019563_c20190730020002.nc> read: 3286409 - 7480713
[INFO: 2020-08-02 16:45:37 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C08_G16_s20190730027190_e20190730029563_c20190730030002.nc
[DEBUG: 2020-08-02 16:45:37 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C08_G16_s20190730027190_e20190730029563_c20190730030002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:38 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C08_G16_s20190730027190_e20190730029563_c20190730030002.nc?generation=1552523418540616&alt=media HTTP/1.1" 206 3284087
[DEBUG: 2020-08-02 16:45:39 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C08_G16_s20190730027190_e20190730029563_c20190730030002.nc> read: 3284087 - 7478391
[INFO: 2020-08-02 16:45:39 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C08_G16_s20190730037190_e20190730039563_c20190730040002.nc
[DEBUG: 2020-08-02 16:45:39 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C08_G16_s20190730037190_e20190730039563_c20190730040002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:39 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C08_G16_s20190730037190_e20190730039563_c20190730040002.nc?generation=1552524050643334&alt=media HTTP/1.1" 206 3281181
[DEBUG: 2020-08-02 16:45:41 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C08_G16_s20190730037190_e20190730039563_c20190730040002.nc> read: 3281181 - 7475485
[INFO: 2020-08-02 16:45:41 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C09_G16_s20190730002190_e20190730004569_c20190730005005.nc
[DEBUG: 2020-08-02 16:45:41 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C09_G16_s20190730002190_e20190730004569_c20190730005005.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:41 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C09_G16_s20190730002190_e20190730004569_c20190730005005.nc?generation=1552521928682875&alt=media HTTP/1.1" 206 3106847
[DEBUG: 2020-08-02 16:45:42 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C09_G16_s20190730002190_e20190730004569_c20190730005005.nc> read: 3106847 - 7301151
[INFO: 2020-08-02 16:45:42 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C09_G16_s20190730027190_e20190730029569_c20190730030002.nc
[DEBUG: 2020-08-02 16:45:42 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C09_G16_s20190730027190_e20190730029569_c20190730030002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:43 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C09_G16_s20190730027190_e20190730029569_c20190730030002.nc?generation=1552523418477429&alt=media HTTP/1.1" 206 3102738
[DEBUG: 2020-08-02 16:45:44 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C09_G16_s20190730027190_e20190730029569_c20190730030002.nc> read: 3102738 - 7297042
[INFO: 2020-08-02 16:45:44 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C09_G16_s20190730037190_e20190730039569_c20190730040002.nc
[DEBUG: 2020-08-02 16:45:44 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C09_G16_s20190730037190_e20190730039569_c20190730040002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:44 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C09_G16_s20190730037190_e20190730039569_c20190730040002.nc?generation=1552524029586334&alt=media HTTP/1.1" 206 3101567
[DEBUG: 2020-08-02 16:45:45 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C09_G16_s20190730037190_e20190730039569_c20190730040002.nc> read: 3101567 - 7295871
[INFO: 2020-08-02 16:45:45 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C10_G16_s20190730002190_e20190730004574_c20190730005012.nc
[DEBUG: 2020-08-02 16:45:45 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C10_G16_s20190730002190_e20190730004574_c20190730005012.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:45 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C10_G16_s20190730002190_e20190730004574_c20190730005012.nc?generation=1552521930264474&alt=media HTTP/1.1" 206 3814035
[DEBUG: 2020-08-02 16:45:47 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C10_G16_s20190730002190_e20190730004574_c20190730005012.nc> read: 3814035 - 8008339
[INFO: 2020-08-02 16:45:47 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C11_G16_s20190730002190_e20190730004563_c20190730005012.nc
[DEBUG: 2020-08-02 16:45:47 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C11_G16_s20190730002190_e20190730004563_c20190730005012.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:47 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C11_G16_s20190730002190_e20190730004563_c20190730005012.nc?generation=1552521928939630&alt=media HTTP/1.1" 206 4694390
[DEBUG: 2020-08-02 16:45:49 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C11_G16_s20190730002190_e20190730004563_c20190730005012.nc> read: 4194304 - 8388608
[DEBUG: 2020-08-02 16:45:49 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C11_G16_s20190730002190_e20190730004563_c20190730005012.nc> read: 4694390 - 8888694
[INFO: 2020-08-02 16:45:49 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C12_G16_s20190730002190_e20190730004569_c20190730005011.nc
[DEBUG: 2020-08-02 16:45:49 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C12_G16_s20190730002190_e20190730004569_c20190730005011.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:50 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C12_G16_s20190730002190_e20190730004569_c20190730005011.nc?generation=1552521927518903&alt=media HTTP/1.1" 206 3825090
[DEBUG: 2020-08-02 16:45:52 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C12_G16_s20190730002190_e20190730004569_c20190730005011.nc> read: 3825090 - 8019394
[INFO: 2020-08-02 16:45:52 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C12_G16_s20190730057190_e20190730059569_c20190730100002.nc
[DEBUG: 2020-08-02 16:45:52 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C12_G16_s20190730057190_e20190730059569_c20190730100002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:53 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C12_G16_s20190730057190_e20190730059569_c20190730100002.nc?generation=1552525223011199&alt=media HTTP/1.1" 206 3798229
[DEBUG: 2020-08-02 16:45:55 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C12_G16_s20190730057190_e20190730059569_c20190730100002.nc> read: 3798229 - 7992533
[INFO: 2020-08-02 16:45:55 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C13_G16_s20190730002190_e20190730004574_c20190730005003.nc
[DEBUG: 2020-08-02 16:45:55 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C13_G16_s20190730002190_e20190730004574_c20190730005003.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:55 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C13_G16_s20190730002190_e20190730004574_c20190730005003.nc?generation=1552521931409553&alt=media HTTP/1.1" 206 4747782
[DEBUG: 2020-08-02 16:45:58 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C13_G16_s20190730002190_e20190730004574_c20190730005003.nc> read: 4194304 - 8388608
[DEBUG: 2020-08-02 16:45:58 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C13_G16_s20190730002190_e20190730004574_c20190730005003.nc> read: 4747782 - 8942086
[INFO: 2020-08-02 16:45:58 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C14_G16_s20190730002190_e20190730004563_c20190730005005.nc
[DEBUG: 2020-08-02 16:45:58 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C14_G16_s20190730002190_e20190730004563_c20190730005005.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:45:58 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C14_G16_s20190730002190_e20190730004563_c20190730005005.nc?generation=1552521929269209&alt=media HTTP/1.1" 206 4754075
[DEBUG: 2020-08-02 16:46:01 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C14_G16_s20190730002190_e20190730004563_c20190730005005.nc> read: 4194304 - 8388608
[DEBUG: 2020-08-02 16:46:01 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C14_G16_s20190730002190_e20190730004563_c20190730005005.nc> read: 4754075 - 8948379
[INFO: 2020-08-02 16:46:01 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C15_G16_s20190730002190_e20190730004569_c20190730005014.nc
[DEBUG: 2020-08-02 16:46:01 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C15_G16_s20190730002190_e20190730004569_c20190730005014.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:46:01 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C15_G16_s20190730002190_e20190730004569_c20190730005014.nc?generation=1552521928370877&alt=media HTTP/1.1" 206 4717193
[DEBUG: 2020-08-02 16:46:04 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C15_G16_s20190730002190_e20190730004569_c20190730005014.nc> read: 4194304 - 8388608
[DEBUG: 2020-08-02 16:46:04 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C15_G16_s20190730002190_e20190730004569_c20190730005014.nc> read: 4717193 - 8911497
[INFO: 2020-08-02 16:46:04 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C16_G16_s20190730002190_e20190730004574_c20190730005014.nc
[DEBUG: 2020-08-02 16:46:04 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C16_G16_s20190730002190_e20190730004574_c20190730005014.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:46:04 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C16_G16_s20190730002190_e20190730004574_c20190730005014.nc?generation=1552521927426861&alt=media HTTP/1.1" 206 3288937
[DEBUG: 2020-08-02 16:46:06 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C16_G16_s20190730002190_e20190730004574_c20190730005014.nc> read: 3288937 - 7483241
[INFO: 2020-08-02 16:46:06 : satpy.demo._google_cloud_platform] Downloading: ./abi_l1b/20190314_us_midlatitude_cyclone/OR_ABI-L1b-RadC-M3C16_G16_s20190730057190_e20190730059575_c20190730100002.nc
[DEBUG: 2020-08-02 16:46:06 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C16_G16_s20190730057190_e20190730059575_c20190730100002.nc> read: 0 - 4194304
[DEBUG: 2020-08-02 16:46:06 : urllib3.connectionpool] https://www.googleapis.com:443 "GET /download/storage/v1/b/gcp-public-data-goes-16/o/ABI-L1b-RadC%2F2019%2F073%2F00%2FOR_ABI-L1b-RadC-M3C16_G16_s20190730057190_e20190730059575_c20190730100002.nc?generation=1552525225519810&alt=media HTTP/1.1" 206 3269973
[DEBUG: 2020-08-02 16:46:08 : fsspec] <File-like object GCSFileSystem, gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/OR_ABI-L1b-RadC-M3C16_G16_s20190730057190_e20190730059575_c20190730100002.nc> read: 3269973 - 7464277
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-5-b9cf2e198143> in <module>
----> 1 filenames = demo.get_us_midlatitude_cyclone_abi()
      2 # This popluates /home/ray/Documents/RCCLWx/data/Satellite/GCP/GOES-16/abi_l1b/

~/local/bin/anaconda3/envs/rcclwx-dev/lib/python3.8/site-packages/satpy/demo/__init__.py in get_us_midlatitude_cyclone_abi(base_dir, method, force)
     95     _makedirs(subdir, exist_ok=True)
     96     filenames = get_bucket_files(patterns, subdir, force=force)
---> 97     assert len(filenames) == 16, "Not all files could be downloaded"
     98     return filenames
     99 

AssertionError: Not all files could be downloaded
```

**Screenshots**

Here's the downloaded files

![image](https://user-images.githubusercontent.com/17162724/89132304-2ca59280-d4e1-11ea-9f9d-c3db48d8172a.png)



**Environment Info:**
 - OS: Linux
 - Satpy Version: [e.g. 0.22.0]
 - PyResample Version: 1.16.0
 - Readers and writers dependencies (when relevant): [run `from satpy.config import check_satpy; check_satpy()`]

Readers
=======
abi_l1b:  ok
abi_l1b_scmi:  ok
abi_l2_nc:  ok
acspo:  ok
agri_l1:  ok
ahi_hrit:  ok
ahi_hsd:  ok
ami_l1b:  ok
amsr2_l1b:  ok
amsr2_l2:  ok
avhrr_l1b_aapp:  ok
avhrr_l1b_eps:  ok
avhrr_l1b_gaclac:  cannot find module 'satpy.readers.avhrr_l1b_gaclac' (No module named 'pygac')
avhrr_l1b_hrpt:  cannot find module 'satpy.readers.hrpt' (No module named 'pygac')
caliop_l2_cloud:  cannot find module 'satpy.readers.caliop_l2_cloud' (No module named 'pyhdf')
clavrx:  cannot find module 'satpy.readers.clavrx' (No module named 'pyhdf')
cmsaf-claas2_l2_nc:  ok
electrol_hrit:  ok
fci_l1c_fdhsi:  ok
fci_l2_nc:  ok
generic_image:  ok
geocat:  ok
ghrsst_l3c_sst:  ok
glm_l2:  ok
goes-imager_hrit:  ok
goes-imager_nc:  ok
gpm_imerg:  ok
grib:  cannot find module 'satpy.readers.grib' (No module named 'pygrib')
hsaf_grib:  cannot find module 'satpy.readers.hsaf_grib' (No module named 'pygrib')
hy2_scat_l2b_h5:  ok
iasi_l2:  ok
iasi_l2_so2_bufr:  cannot find module 'satpy.readers.iasi_l2_so2_bufr' (('Missing eccodes-python and/or eccodes C-library installation. Use conda to install eccodes.\n           Error: ', ModuleNotFoundError("No module named 'eccodes'")))
jami_hrit:  ok
li_l2:  cannot find module 'satpy.readers.li_l2' (No module named 'h5netcdf')
maia:  ok
mersi2_l1b:  ok
mimicTPW2_comp:  ok
modis_l1b:  cannot find module 'satpy.readers.modis_l1b' (No module named 'pyhdf')
modis_l2:  cannot find module 'satpy.readers.modis_l2' (No module named 'pyhdf')
msi_safe:  cannot find module 'satpy.readers.msi_safe' (No module named 'glymur')
mtsat2-imager_hrit:  ok
nucaps:  ok
nwcsaf-geo:  ok
nwcsaf-msg2013-hdf5:  ok
nwcsaf-pps_nc:  ok
olci_l1b:  ok
olci_l2:  ok
omps_edr:  ok
safe_sar_l2_ocn:  ok
sar-c_safe:  ok
scatsat1_l2b:  ok
seviri_l1b_hrit:  ok
seviri_l1b_icare:  cannot find module 'satpy.readers.seviri_l1b_icare' (No module named 'pyhdf')
seviri_l1b_native:  ok
seviri_l1b_nc:  ok
seviri_l2_bufr:  cannot find module 'satpy.readers.seviri_l2_bufr' (Missing eccodes-python and/or eccodes C-library installation. Use conda to install eccodes)
seviri_l2_grib:  cannot find module 'satpy.readers.seviri_l2_grib' (Missing eccodes-python and/or eccodes C-library installation. Use conda to install eccodes)
slstr_l1b:  ok
slstr_l2:  ok
smos_l2_wind:  ok
tropomi_l2:  ok
vaisala_gld360:  ok
vii_l1b_nc:  ok
vii_l2_nc:  ok
viirs_compact:  ok
viirs_edr_active_fires:  ok
viirs_edr_flood:  cannot find module 'satpy.readers.viirs_edr_flood' (No module named 'pyhdf')
viirs_l1b:  ok
viirs_sdr:  ok
virr_l1b:  ok

Writers
=======
cf:  ok
geotiff:  ok
mitiff:  ok
ninjotiff:  cannot find module 'satpy.writers.ninjotiff' (No module named 'pyninjotiff')
scmi:  ok
simple_image:  ok

Extras
======
cartopy:  ok
geoviews:  No module named 'geoviews'

**Additional context**

The functions expects 16 files to be downloaded
https://github.com/pytroll/satpy/blob/master/satpy/demo/__init__.py#L97

--- a/satpy/demo/__init__.py
+++ b/satpy/demo/__init__.py
@@ -56,18 +56,6 @@ import logging
 LOG = logging.getLogger(__name__)
 
 
-def _makedirs(directory, exist_ok=False):
-    """Python 2.7 friendly os.makedirs.
-
-    After python 2.7 is dropped, just use `os.makedirs` with `existsok=True`.
-    """
-    try:
-        os.makedirs(directory)
-    except OSError:
-        if not exist_ok:
-            raise
-
-
 def get_us_midlatitude_cyclone_abi(base_dir='.', method=None, force=False):
     """Get GOES-16 ABI (CONUS sector) data from 2019-03-14 00:00Z.
 
@@ -90,16 +78,16 @@ def get_us_midlatitude_cyclone_abi(base_dir='.', method=None, force=False):
                                   "implemented yet.".format(method))
 
     from ._google_cloud_platform import get_bucket_files
-    patterns = ['gs://gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/*0002*.nc']
+    patterns = ['gs://gcp-public-data-goes-16/ABI-L1b-RadC/2019/073/00/*s20190730002*.nc']
     subdir = os.path.join(base_dir, 'abi_l1b', '20190314_us_midlatitude_cyclone')
-    _makedirs(subdir, exist_ok=True)
+    os.makedirs(subdir, exist_ok=True)
     filenames = get_bucket_files(patterns, subdir, force=force)
     assert len(filenames) == 16, "Not all files could be downloaded"
     return filenames
 
 
 def get_hurricane_florence_abi(base_dir='.', method=None, force=False,
-                               channels=range(1, 17), num_frames=10):
+                               channels=None, num_frames=10):
     """Get GOES-16 ABI (Meso sector) data from 2018-09-11 13:00Z to 17:00Z.
 
     Args:
@@ -122,6 +110,8 @@ def get_hurricane_florence_abi(base_dir='.', method=None, force=False,
     Total size (240 frames, all channels): ~3.5GB
 
     """
+    if channels is None:
+        channels = range(1, 17)
     if method is None:
         method = 'gcsfs'
     if method not in ['gcsfs']:
@@ -145,7 +135,7 @@ def get_hurricane_florence_abi(base_dir='.', method=None, force=False,
             'gs://gcp-public-data-goes-16/ABI-L1b-RadM/2018/254/16/*RadM1*C{:02d}*s201825416*.nc'.format(channel),
         )]
     subdir = os.path.join(base_dir, 'abi_l1b', '20180911_hurricane_florence_abi_l1b')
-    _makedirs(subdir, exist_ok=True)
+    os.makedirs(subdir, exist_ok=True)
     filenames = get_bucket_files(patterns, subdir, force=force, pattern_slice=frame_slice)
 
     actual_slice = frame_slice.indices(240)  # 240 max frames

